### PR TITLE
Update branding to 8.0.7

### PR DESCRIPTION
**Pull request by vseanreesermsft**

### CODE REVIEW

File: eng/Versions.props [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
     <!-- The .NET product branding version -->
-    <ProductVersion>8.0.6</ProductVersion>
+    <ProductVersion>8.0.7</ProductVersion>
     <!-- File version numbers -->
     <MajorVersion>8</MajorVersion>
     <MinorVersion>0</MinorVersion>
-    <PatchVersion>6</PatchVersion>
+    <PatchVersion>7</PatchVersion>
     <SdkBandVersion>8.0.100</SdkBandVersion>
     <PackageVersionNet7>7.0.19</PackageVersionNet7>
```
Comment (lewing) [inline eng/Versions.props:10]:
```suggestion
    <PackageVersionNet7>7.0.20</PackageVersionNet7>
```

**Review by lewing — COMMENTED** [review]
we're going to need to resolve things for the downlevel versions

File: eng/Versions.props (hunk 1 above)
Comment (lewing) [reply eng/Versions.props:10]: this likely applies to 8.0.6 too

**Review by carlossanlop — APPROVED** [review]

**Comment by carlossanlop** [comment]
@lewing lmk if this is good to go

**Review by lewing — APPROVED** [review]

**Pull request merged**